packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Checkbox from './Checkbox';
+import { applyTheme } from 'cf-style-container';
 import includes from 'lodash/includes';
 
+import CheckboxUnstyled from './Checkbox';
+import CheckboxTheme from './CheckboxTheme';
+
+const Checkbox = applyTheme(CheckboxUnstyled, CheckboxTheme, theme => ({
```

Comment:
Hm, are there any other props that you would need to add? Giving the component some contextual awareness through an extra prop (in other words telling it "you are in a group") seems reasonable to me. It worked out pretty well for the Button.